lib/farm_ng/oak/__init__.py

```diff
@@ -0,0 +1 @@
+from .oak_client import *  # noqa: F401, F403
```

Comment:
Maybe we shouldn't do this type of import? Or we should have this in our style policy. I personally don't like this as a user as I can't tell where the import came from when tracing code. I don't mind typing out the full import path.

Reply:
I'm open to anything as soon as we decide a good name structure. Want to avoid to from farm_ng.oak.oak_client import OakClient